Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -190,7 +190,7 @@ static CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localiz
     NSDictionary* attribs = [ts->_attributedString attributesAtIndex:range.location effectiveRange:NULL];
 
     CGFloat fontSize = kCTFontSystemFontSize;
-    std::vector<wchar_t> familyName(c_defaultFontName, c_defaultFontName + wcslen(c_defaultFontName));
+    std::vector<wchar_t> familyName;
```

Comment:
>familyName [](start = 25, length = 10)

if we do not initialize it here and we have no font we will try to get the address in the CreateTextFormat call, which I imagine should cause some issues.